README.md

```diff
@@ -230,3 +230,5 @@ const customizedFooterProp: IFooterProps = {
 [Omnichannel Features](https://github.com/microsoft/omnichannel-chat-widget/blob/main/docs/Features.md)\
 [How to Add Visual Regression Tests](https://github.com/microsoft/omnichannel-chat-widget/blob/main/docs/VisualRegressionTestingGuide.md)\
 [Security](https://github.com/microsoft/omnichannel-chat-widget/blob/main/SECURITY.md)
+
```

Comment:
I think empty line is not always supported on other platforms. Please change to use "\" as the others above